### PR TITLE
LPS-119569 Feedback messages are not standardized

### PR DESCRIPTION
**Pull request by Valmir-unicap**
Related issue: https://issues.liferay.com/browse/LPS-119569

### CODE REVIEW

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:relevant

**Comment by Valmir-unicap** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -6,6 +6,16 @@
...
+.form-control[type='text'] {
+	background-color: #e7e7ed;
+}
+.has-error .form-control[readonly] {
+	background-color: #e7e7ed;
+}
+.has-success .form-control[readonly] {
+	background-color: #e7e7ed;
+}
+
```
Comment (alinedoleron) [inline modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/resources/META-INF/resources/css/main.scss:18]: Hi @Valmir-unicap,

please, could you make this adjustments? This keep the red colors related to the field errors. Thanks!

```suggestion
.form-control[readonly] {
	background-color: #f1f2f5;
        border-color: #e7e7ed;
	cursor: not-allowed;
}

.has-success .form-control[type='text'] {
	background-color: #f1f2f5;
	border-color: #e7e7ed;
}

.has-error .form-control[readonly] {
	background-color: #feefef;
        border-color: #f48989;
}

```

**Review by alinedoleron — CHANGES_REQUESTED** [review]

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/resources/META-INF/resources/css/main.scss (hunk 1 above)
Comment (Valmir-unicap) [reply modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/resources/META-INF/resources/css/main.scss]: Corrected

Comment (Valmir-unicap) [reply modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/resources/META-INF/resources/css/main.scss]: Thank you :)

**Review by alinedoleron — APPROVED** [review]
LGTM!

**Comment by alinedoleron** [comment]
ci:forward

**Comment by alinedoleron** [comment]
ci:forward

**Pull request closed without merging**